content/en/database_monitoring/setup_postgres/aurora.md

```diff
@@ -112,7 +112,7 @@ SECURITY DEFINER;
 {{% /tab %}}
 {{< /tabs >}}
 
-**Note**: When generating custom metrics that require querying additional tables, you may need to grant the `SELECT` permission on those tables to the `datadog` user. Example: `grant SELECT on <TABLE_NAME> to datadog;`. See [PostgreSQL custom metric collection explained][6] for more information.
+<div class="alert alert-info">For metric/data collection or custom metrics that require querying additional tables, you may need to grant the <code>SELECT</code> permission on those tables to the datadog user. Example: <code>grant SELECT on &lt;TABLE_NAME&gt; to datadog;</code>. See <a href="https://docs.datadoghq.com/integrations/faq/postgres-custom-metric-collection-explained//">PostgreSQL custom metric collection</a> for more information. </div>
```

Comment:
```suggestion
<div class="alert alert-info">For data collection or custom metrics that require querying additional tables, you may need to grant the <code>SELECT</code> permission on those tables to the <code>datadog</code> user. Example: <code>grant SELECT on &lt;TABLE_NAME&gt; to datadog;</code>. See <a href="https://docs.datadoghq.com/integrations/faq/postgres-custom-metric-collection-explained/">PostgreSQL custom metric collection</a> for more information. </div>
```

Reply:
Just going to leave this one suggestion since the update is identical across the files. The only important update is to remove the extra slash from the URL since that's breaking it -- other updates are optional, but I think uncapitalized datadog looks weird when not being rendered as a code snippet or in a code block (just my perspective).